sched/scheduler/cluster_state.go

```diff
@@ -225,15 +226,15 @@ func (c *clusterState) update(updates []cluster.NodeUpdate) {
 			if ns, ok := c.suspendedNodes[update.Id]; ok {
 				if !ns.ready() {
 					// Adding a node that's already suspended as non-ready, leave it in that state until ready.
-					log.Infof("Suspended node re-added but still awaiting readiness check %v (%#v)", update.Id, ns)
+					log.Infof("Suspended node re-added but still awaiting readiness check %v (%s)", update.Id, ns)
```

Comment:
And what was the reason for going from #%v to %s? If I remember correctly, I changed this so it would print the field names rather than just bare values.

Reply:
using %s trigger using ns.String() which prints the host name, shard id as well as the IP address from update.Node.  ns.String() prints field name as well as values.  The big difference is that it does not print readyCh nor removedCh.
You can see the differences if you compare devel vs prod loglens entries (for 1/24 noon to 1pm'ish)